packages/fabric8-ui/src/app/dashboard-widgets/space-settings-widget/space-settings-widget.component.html

```diff
@@ -0,0 +1,130 @@
+<f8-feature-toggle
+  featureName="Analyze.WorkItemOverviewCard"
+  [userLevel]="user"
+></f8-feature-toggle>
+<ng-template #user>
+  <div id="spacehome-workitems-card" class="space-settings-widget card-pf f8-small-card">
+    <div class="card-pf-heading f8-card-heading"><h2 class="card-pf-title">Space settings</h2></div>
+    <div class="card-pf-body f8-small-card-body">
+      <div class="">
+        <p>{{ UserEmail }} ({{ UserName }})</p>
+        <ul class="f8-space-settings-data-links">
+          <li>
+            <span class="pficon pficon-info pficon-users fa-sm"> </span>
+            {{ collaboratorCount }} Collaborators |
```

Comment:
As per the UX, this should be a link. But I do not whether it will take the user to collaborators page or open a pop-up to show the collaborators name.

Reply:
We do not have the route and page yet to add the links for collaborators, areas and environments.